x-pack/plugins/lens/public/xy_visualization/xy_config_panel/annotations_config_panel/index.scss

```diff
@@ -0,0 +1,12 @@
+.lnsRowCompressedMargin+.lnsRowCompressedMargin {
+  margin-top: $euiSizeS;
+}
+
+.lnsConfigPanelNoPadding {
+  padding: 0;
+}
+
+.lnsConfigPanelDate__label {
+  min-width: 56px;
```

Comment:
After thinking about it a bit I think this is the best solution in this particular scenario. However a value like this looks like an immediate code smell, could we add a comment to explain?